docs/source/index.rst

```diff
@@ -2,17 +2,38 @@
 opt_einsum
 ==========
 
-:func:`~numpy.einsum` is a powerful function for contracting tensors of arbitrary
-dimension and index. However, it is only optimized to contract two terms
-at a time resulting in non-optimal scaling.
+Optimized einsum can greatly reduce the overall execution time of einsum-like
+expressions by optimizing the expression's contraction order and dispatching
+many operations to canonical BLAS, cuBLAS, or other specialized routines.
+Optimized einsum is agnostic to the backend and can handle NumPy, Dask,
+PyTorch, Tensorflow, CuPy, Sparse, Theano, Jax, and Autograd arrays.
 
-For example, consider the following index transformation:
+Features
+========
+
+The algorithms found in this repository often power the ``einsum`` optimizations
+in many of the above projects. For example, the optimization of ``np.einsum``
+has been passed upstream and most of the same features that can be found in
+this repository can be enabled with ``numpy.einsum(..., optimize=True)``. However,
+this repository often has more up to date algorithms for complex contractions.
+Several advanced features are as follows:
+
+* Inspect `detailed information <path_finding.html>`_ about the path chosen.
+* Perform contractions with `numerous backends <backends.html>`_, including on the GPU and with libraries such as `TensorFlow <https://www.tensorflow.org>`_ and `PyTorch <https://pytorch.org>`_.
+* Generate `reusable expressions <reusing_paths.html>`_, potentially with `constant tensors <reusing_paths.html#specifying-constants>`_, that can be compiled for greater performance.
+* Use an arbitrary number of indices to find contractions for `hundreds or even thousands of tensors <ex_large_expr_with_greedy.html>`_.
+* Share `intermediate computations <sharing_intermediates.html>`_ among multiple contractions.
+
```

Comment:
Compared to this same list elsewhere, the bullet point regarding autograd/jax is missing.